test/cmd/quota.sh

```diff
@@ -4,6 +4,16 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/quota"
 
+# Cleanup cluster resources created by this test suite
```

Comment:
I understand why we'd want to wait for the namespaces to get cleaned up, but I think we should not do it because:
 - nowhere else do we bother
 - whoever is using the tests in that manner (existing cluster, rerun suite) hasn't complained yet
 - I want to kill `wait_for_command`, not increase its usage.